config/initializers/ezid.rb

```diff
@@ -0,0 +1,7 @@
+unless Rails.env.production?
```

Comment:
Maybe we should remove this and just make sure the EZID stuff is configured in environment variables for staging/production?